### PR TITLE
Remove ECR auth and document gotcha with iam service account

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 1]
```diff
@@ -148,23 +148,29 @@ eksctl create iamserviceaccount \
...
 
-== Install Akka Operator
+To delete run:
 
-. Authenticate docker to the Amazon ECR repository for Akka Cloud Platform:
-+
-[source,shell script]
+[source, shell script]
 ----
-aws ecr get-login-password --region eu-central-1 | docker login --username AWS --password-stdin 709825985650.dkr.ecr.us-east-1.amazonaws.com
```
Comment (chbatey) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:165]: This was here from when we had instructions to push the image.

This doesn't work, you can't pull the image locally. Only an EKS cluster can pull it.

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 2]
```diff
@@ -148,23 +148,29 @@ eksctl create iamserviceaccount \
...
     --namespace lightbend \
     --cluster eks-akka-demo \
-    --region eu-central-1
+    --region eu-central-1 \
     --attach-policy-arn <policy_arn> \
     --approve \
     --override-existing-serviceaccounts
 ----
 
+If this fails for any reason it is important to delete the service account before trying to re-create otherwise `eksctl` can skip creation
```
Comment (chbatey) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:157]: This is very annoying eksctl behavior

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 3]
```diff
@@ -148,23 +148,29 @@ eksctl create iamserviceaccount \
...
-== Install Akka Operator
+To delete run:
 
-. Authenticate docker to the Amazon ECR repository for Akka Cloud Platform:
-+
-[source,shell script]
+[source, shell script]
 ----
-aws ecr get-login-password --region eu-central-1 | docker login --username AWS --password-stdin 709825985650.dkr.ecr.us-east-1.amazonaws.com
+eksctl delete iamserviceaccount akka-operator
```
Comment (patriknw) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:168]: include `--region eu-central-1` ?

Comment (chbatey) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:168]: ah yes

**Pull request merged**